src/libraries/System.Web.HttpUtility/tests/HttpUtility/HttpUtilityTest.cs

```diff
@@ -284,6 +284,20 @@ public void HtmlEncode_TextWriter_null()
             });
         }
 
+        [Fact]
+        public void HtmlEncode_IHtmlString_UseToHtmlString()
+        {
+            Assert.Equal(string.Empty, HttpUtility.HtmlEncode(new ActionHtmlString(() => null)));
+            Assert.Equal(string.Empty, HttpUtility.HtmlEncode(new ActionHtmlString(() => string.Empty)));
+            Assert.Equal("42", HttpUtility.HtmlEncode(new ActionHtmlString(() => "42")));
```

Comment:
```suggestion
            Assert.Equal("<", HttpUtility.HtmlEncode(new ActionHtmlString(() => "<")));
```
To test that we're skipping the encoding.